pkg/cli/computeclasses.go

```diff
@@ -0,0 +1,47 @@
+package cli
```

Comment:
Didn't we agree that this should move to `acorn offerings computeclasses`?

Reply:
Yep! Sorry about that, looks like I pushed up an old stash around this.

Reply:
Fixed!